gnat2goto/driver/driver.adb

```diff
@@ -435,59 +438,57 @@ package body Driver is
                      Arg_Type : constant Irep := Get_Type (Arg);
                      Arg_Id   : constant Symbol_Id :=
                        Intern ("input_" &  Get_Identifier (Arg));
-                     Arg_Symbol : Symbol;
+                     Arg_Symbol : constant Symbol :=
```

Comment:
There is a wrapper for creating function arguments in `goto_utils`: `Create_Fun_Parameter` and `Param_Symbol`.

Reply:
Hm, I think I prefer this still